### ISSUE TITLE
Two Rultor messages on merge closed pull request

### ISSUE THREAD

**Issue by aistomin** [yegor256/rultor#1007]
I've made some mistakes when was talking to Rultor and noticed a behaviour that is not clear to me. Maybe it's an expected behaviour, would be nice if somebody could clarify it to me.

Pull request where problem occurred: https://github.com/aistomin/jenkins-sdk/pull/182

Questions:
1. I sent Rultor two consiquent merge messages by mistake, why didn't Rultor just ignore the second message?
2. Ok, Rultor interpreted the second message as a command inside already merged PR, why he sent me two error messages "The pull request is closed already, so I can't merge it" and "I'm sorry, I don't understand you :( ". One message I think is enough :)

**Comment by alex-palevsky** [comment]
@yegor256 please pay attention to this issue ([par.21](http://at.teamed.io/policy.html#21))

**Comment by alex-palevsky** [comment]
@aistomin since there is no milestone yet I set it to "2.0"

**Comment by original-brownbear** [comment]
@alex-palevsky this depends on #917.

**Comment by alex-palevsky** [comment]
> @alex-palevsky this depends on #917.

@original-brownbear ok, let's wait for #917

**Comment by original-brownbear** [comment]
@alex-palevsky this is postponed.

**Comment by alex-palevsky** [comment]
> @alex-palevsky this is postponed.

@original-brownbear right, I added "postponed" label

**Comment by alex-palevsky** [comment]
> @alex-palevsky this is postponed.

@original-brownbear right, I'll try to find someone else for this task
